recipe/build.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/sh
 
-libtoolize --copy
-autoreconf -vfi
-
-./configure          \
-  --prefix=${PREFIX} \
-  --enable-static    \
-  --enable-shared    \
-  --without-octave   \
-  --without-matlab   \
-  --without-guile
-
-make -j${CPU_COUNT} ${VERBOSE_AT}
-make install ${VERBOSE_AT}
+if [ "$(uname)" = "Darwin" ]; then
+  # osx uses a temp directory which messes up with cmake finding libraries.
+  cd "${SRC_DIR}" || exit
+fi
+
+mkdir build && cd build || exit
```

Comment:
Bash scripts should have `set -euo pipeline` at the start to handle commands that fail automatically.
(For existing scripts, like `r-base`'s `build.sh`, the `-u` flag may not be welcome.)

```suggestion

set -euo pipeline

if [ "$(uname)" = "Darwin" ]; then
  # osx uses a temp directory which messes up with cmake finding libraries.
  cd "${SRC_DIR}"
fi

mkdir build
cd build

```

Reply:
Fixed

Reply:
`set -euo pipefail`